pages/api/tags/get-all-tags.js

```diff
@@ -0,0 +1,46 @@
+import * as Utilities from "~/node_common/utilities";
```

Comment:
mind renaming this file to get-user-tags? I think in the future it's possible we'll have a separate get all tags (like get all tags across all of slate) and this might be confused for that

Reply:
This endpoint is not need again